wvd-sh/WVD scaling script/basicScale.ps1

```diff
@@ -305,17 +294,27 @@ if ($hostpoolInfo.LoadBalancerType -eq "DepthFirst") {
                 if ($numberOfRunningHost -lt $MinimumNumberOfRDSH) {
                     $hostsessions = $sessionHost.Sessions
                     if ($hostpoolMaxSessionLimit -ne $hostsessions) {
-                        if ($sessionhost.Status -eq "NoHeartbeat") {
-                            $sessionhostname = $sessionhost.sessionhostname
+						#Check the session host status and session host is healthy or not before starting the host																						  
```

Comment:
All of the same changes from "basicScale-MSI.ps1" applies to "basicScale.ps1"

Reply:
Thank you Christian for to addressing us. We have checked and updated in both scripts